mkdocs.yml

```diff
@@ -138,7 +172,8 @@ nav:
         - 'Accumulo Operations': 'reference/operations-guide/accumulo.md'
         - 'Spark Operations': 'reference/operations-guide/spark.md'
         - 'Misc Operations': 'reference/operations-guide/misc.md'
-
+      - Endpoints:
```

Comment:
Not sure if its worth having this under a folder probably only going to be one page so could be just following instead like with javadoc and glossary:
`Endpoints: 'reference/endpoints.md'`